include/ultra64/message.h

```diff
@@ -36,4 +36,11 @@ typedef struct OSMesgQueue {
     /* 0x14 */ OSMesg* msg;
 } OSMesgQueue; // size = 0x18
 
+/* Get count of valid messages in queue */
+#define MQ_GET_COUNT(mq)    ((mq)->validCount)
+
+/* Determine if message queue is empty or full */
+#define MQ_IS_EMPTY(mq)     (MQ_GET_COUNT(mq) == 0)
+#define MQ_IS_FULL(mq)      (MQ_GET_COUNT(mq) >= (mq)->msgCount)
+
```

Comment:
This might be fine as is due to context, but I worry if `MQ` might be conflated with "Master Quest" instead of "Message Queue". Is there any downside to having the prefix start with `MESQ_`?

Reply:
I don't think we should make any breaking changes to sdk headers. I'd imagine context is sufficient here

Reply:
If `MQ_` is an already preexisting standard already outside of decomp, then yeah I suppose it's fine

Reply:
This did cross my mind, but I hope we can rely on people to tell the difference.

Reply:
I think context is enough, but I'm curious why we would want to keep the original headers intact?

Reply:
It seems simply better to be compatible than not, also means any sdk documentation stays immediately applicable.

Reply:
I've done this in libultra cleanup 3 ™ but figured I'd do it for this file while I was here. I may split it into it's own PR though.